pkgs/development/libraries/libbladeRF/default.nix

```diff
@@ -10,24 +10,15 @@ let
   };
 in stdenv.mkDerivation rec {
   pname = "libbladeRF";
-  version = "2.2.1";
+  version = "2.4.1";
```

Comment:
Is the `no-OS` (top of the file) version still okay, or does that also require a version bump?

Reply:
It still works for me, so I decided not to touch it. The commit in nixpkgs right now refers to a 2017 release, while the latest release is from 2019. I suppose I could bump it as well, but that would require more testing that I might not be able to provide.